requirements/requirements_ansible.txt

```diff
@@ -103,21 +103,21 @@ pyparsing==2.4.0          # via packaging
 python-dateutil==2.6.1    # via adal, azure-storage, botocore
 pyvmomi==6.5
 pywinrm[kerberos]==0.3.0
-pyyaml==5.1               # via azure-cli-core, knack, openstacksdk
+pyyaml==5.1.1             # via azure-cli-core, knack, openstacksdk
```

Comment:
I've been trying to add another sentence to the instructions, but it got removed along with some other unrelated churn:

> If you do not wish to upgrade dependencies generally (such as adding a specific dependency needed for a feature) then do not use the `-U` flag.

We do need to upgrade dependencies wholesale and test with the new stuff, but for minor changes like what you're doing, it does not make sense. I hope that will help reduce the magnitude of what you're trying to accomplish with this diff.